src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -223,49 +214,43 @@ suspend fun blockingApplicationLogic(
                         keyValue("hendelsesId", hendelsesId)
                     )
 
-                    log.info("Received paper sicklave, $logKeys", *logValues)
-
-                    log.info("Calling saf graphql on journalpostid: ${journalfoeringHendelseRecord.journalpostId}")
-                    val journalpost = safJournalpostClient.getJournalpostMetadata(
-                        journalfoeringHendelseRecord.journalpostId.toString()
-                    )!!
+                    log.info("Received papirsykmelding, $logKeys", *logValues)
+                    val journalpost = safJournalpostClient.getJournalpostMetadata(journalpostId)!!
 
-                    log.info("Svar far saf graphql om journalpost Bruker: ${journalpost.bruker()!!.id()!!}")
+                    log.debug("Response from saf graphql $logKeys", *logValues)
 
-                    val aktoerIdPasient = findAktorid(journalpost, aktoerIdClient, sykmeldingId, credentials, logKeys, logValues, journalpostId)
-                    val fnrPasient = findFNR(journalpost, aktoerIdClient, sykmeldingId, credentials, logKeys, logValues, journalpostId)
+                    val aktoerIdPasient = findAktorid(journalpost, aktoerIdClient, sykmeldingId, credentials)
+                    val fnrPasient = findFNR(journalpost, aktoerIdClient, sykmeldingId, credentials)
 
                     val geografiskTilknytning = fetchGeografiskTilknytning(personV3, fnrPasient)
                     val patientDiskresjonsKode = fetchDiskresjonsKode(personV3, fnrPasient)
-                    val finnBehandlendeEnhetListeResponse =
-                        fetchBehandlendeEnhet(arbeidsfordelingV1, geografiskTilknytning.geografiskTilknytning, patientDiskresjonsKode)
+                    val finnBehandlendeEnhetListeResponse = fetchBehandlendeEnhet(arbeidsfordelingV1,
+                            geografiskTilknytning.geografiskTilknytning, patientDiskresjonsKode)
 
-                    if (finnBehandlendeEnhetListeResponse?.behandlendeEnhetListe?.firstOrNull()?.enhetId == null) {
-                        log.error("arbeidsfordeling fant ingen nav-enheter $logKeys", *logValues)
+                    val behandlerEnhetsId = finnBehandlendeEnhetListeResponse?.behandlendeEnhetListe?.firstOrNull()?.enhetId ?: let {
+                        log.error("Unable to find a NAV enhet, defaulting to $STANDARD_NAV_ENHET $logKeys", *logValues)
+                        STANDARD_NAV_ENHET
                     }
 
-                    val sakid = findSakid(sakClient, sykmeldingId, aktoerIdPasient, logKeys, logValues)
+                    val sakId = findSakid(sakClient, sykmeldingId, aktoerIdPasient, logKeys, logValues)
```

Comment:
Det gjøre mye forretningslogikk i Bootstrap her. Jeg tenker det hører hjemme i en form for behandlingsService slik at vi kan skille mellom litt mer hardt oppsett (kafka topics, ette opp apier osv) og forretningslogikk(innhente data og opprette oppgave)